src/vue/backend/src/templates/GuidedSetupContentCard.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+    <idx-dialog :show="showDialog" @dismiss="closeDialog" customClass="gs-dialog">
+        <template v-slot:header>
+            <idx-block className="dialog-header">
+                <idx-block className="dialog-header__title">{{ title }}</idx-block>
+                <idx-block className="dialog-header__dismiss">
+                    <span @click="closeDialog">Close X</span>
+                </idx-block>
+            </idx-block>
+        </template>
+        <ContentCard :steps="steps" :cardTitle="cardTitle" :relatedLinks="relatedLinks">
+            <template v-slot:description></template>
```

Comment:
This will pass empty information to the content card, lets chat about possibly combining the two components to avoid this or passing slot information.